### PR TITLE
Add island mode status

### PR DESCRIPTION
**Pull request by MichaelMMS**
Get info about the island mode status.

### CODE REVIEW

File: src/rctmon/models.py [hunk 1]
```diff
@@ -357,6 +359,11 @@ def collect_gen(gen: SolarGeneratorReadings, name: str, gen_name: str) -> None:
             ivs.add_metric([name], self.inverter_status)
             yield ivs
 
+        if self.inverter_grid_separated is not None:
+            igs = GaugeMetricFamily('rctmon_inverter_grid_separated', 'Status of the island mode', labels=['inverter', 'grid'])
```
Comment (svalouch) [inline src/rctmon/models.py]: You're specifying that it requires two labels, but it only needs one (`inverter` name) and you're only giving it the one in the next line, so this should be `labels=['inverter']`.

**Review by svalouch — CHANGES_REQUESTED** [review]
Thanks and sorry for the delay. There's a slight problem with the labels, otherwise this looks fine!

File: src/rctmon/models.py (hunk 1 above)
Comment (MichaelMMS) [reply src/rctmon/models.py]: Thx, I removed grid label

**Review by MichaelMMS — COMMENTED** [review]
Thx, I removed grid label

**Comment by svalouch** [comment]
Thank you!

**Pull request merged**